trie/database.go

```diff
@@ -847,3 +852,46 @@ func (db *Database) Size() (common.StorageSize, common.StorageSize) {
 	var metarootRefs = common.StorageSize(len(db.dirties[common.Hash{}].children) * (common.HashLength + 2))
 	return db.dirtiesSize + db.childrenSize + metadataSize - metarootRefs, db.preimagesSize
 }
+
+func (db *Database) saveCache(dir string, threads int) (err error) {
+	if db.cleans == nil {
+		return
+	}
+	defer func(start time.Time) {
+		if err == nil {
+			log.Info("Saved clean cache into the file", "path", dir, "elapsed", common.PrettyDuration(time.Since(start)))
+		} else {
+			log.Info("Failed to save clean cache into file", "error", err)
+		}
+	}(time.Now())
+	err = db.cleans.SaveToFileConcurrent(dir, threads)
```

Comment:
Is there a reason why we'd not want to use max concurrency here? I'd just set this to `0` and let `fastcache` max out the performance.

Reply:
How much time does it take to persist a 1GB cache?

Reply:
I can't remember the concrete number, but a few seconds.